Fakebook.Posts/Fakebook.Posts.DataAccess/Models/FakebookPostsContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Fakebook.Posts.DataAccess.Models {
```

Comment:
Does it make sense for the context to be in Models namespace?
Shouldn't the context import the models namespace?
The rest of the code we wrote assumes the context is in the DataAccess namespace.

Reply:
dbcontext has been moved outside of the models folder, namespace has been updated.